Content.Server/Blocking/BlockingSystem.cs

```diff
@@ -0,0 +1,72 @@
+using Content.Server.Examine;
+using Content.Shared.Blocking;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Verbs;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Blocking;
```

Comment:
I don't see the need to make a server class. You can access GetVerbsEvent from shared. Please revert this change and add what you did here into Shared instead.